cypress/support/commands.ts

```diff
@@ -246,3 +246,190 @@ Cypress.Commands.add("unlock", () => {
+
+    // Give the app some time to finish animations
+    cy.wait(100);
+
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-input#emailInput")
+        .find("input")
+        .type(email, { force: true });
+
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-input#nameInput")
+        .find("input")
+        .type(name, { force: true });
+
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-loading-button#submitEmailButton")
+        .click({ force: true });
+
+    // Give the app some time to finish animations
+    cy.wait(100);
+
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-input#codeInput")
+        .find("input")
+        .type(emailToken, { force: true });
+
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-loading-button#verifyEmailButton")
+        .click({ force: true });
+
+    // Give the app some time to finish animations
+    cy.wait(100);
+
+    // Choose a different password
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("div.wrapper:eq(2) div.password-actions button:eq(1)")
+        .click({ force: true });
+
+    // Give the app some time to finish animations
+    cy.wait(100);
+
+    // Choose my own
+    cy.get("pl-app").find("pl-alert-dialog").find("button:eq(2)").click({ force: true });
+
+    // Give the app some time to finish animations
+    cy.wait(200);
+
+    // Type master password
+    cy.get("pl-app").find("pl-prompt-dialog").find("pl-input.tap").find("input").type(password, { force: true });
+
+    // Confirm master password
+    cy.get("pl-app").find("pl-prompt-dialog").find("pl-loading-button#confirmButton").click({ force: true });
+
+    // Give the app some time to render the alert, otherwise it sometimes shows out of place
+    cy.wait(200);
+
+    // Confirm weak password
+    cy.get("pl-app").find("pl-alert-dialog").find("button:eq(1)").click({ force: true });
+
+    // Give the app some time to finish animations
+    cy.wait(100);
+
+    // Repeat master password
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("div.wrapper:eq(2) pl-password-input#repeatPasswordInput")
+        .find("input[type='password']")
+        .type(password, { force: true });
+
+    // Continue signup
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-signup")
+        .find("pl-loading-button#submitPasswordButton")
+        .click({ force: true });
+
+    // Wait for success
+    cy.url().should("include", "/signup/success");
+
+    // Done!
+    cy.get("pl-app")
+        .find("pl-start")
+        .find("pl-login-signup")
+        .find("pl-drawer:eq(7)")
+        .find("pl-button")
+        .click({ force: true });
+
+    cy.url().should("include", "/items");
```

Comment:
This part is still "untested" (and it won't work since there's no `pl-login-signup`) because I can't get past the `srp` error.